cli/pcluster/configure/easyconfig.py

```diff
@@ -0,0 +1,424 @@
+    )
+
+    config = configparser.ConfigParser()
+    # Check if configuration file exists
+    if os.path.isfile(config_file):
+        config.read(config_file)
+
+    # Prompt for required values, using existing as defaults
+    cluster_template = DEFAULT_VALUES["cluster_template"]
+    cluster_label = "cluster " + cluster_template
+    vpc_label = "vpc " + cluster_template
+
+    # Use built in boto regions as an available option
+    aws_region_name = prompt_iterable(
+        "AWS Region ID",
+        get_regions(),
+        default_value=_get_config_parameter(
+            config, section="aws", parameter_name="aws_region_name", default_value=DEFAULT_VALUES["aws_region_name"]
+        ),
+    )
+
+    scheduler = prompt_iterable(
+        "Scheduler",
+        get_supported_schedulers(),
+        default_value=_get_config_parameter(
+            config, section=cluster_label, parameter_name="scheduler", default_value=DEFAULT_VALUES["scheduler"]
+        ),
+    )
+
+    scheduler_handler = SchedulerHandler(config, cluster_label, scheduler)
+
+    scheduler_handler.prompt_os()
+    scheduler_handler.prompt_cluster_size()
+
+    master_instance_type = prompt(
+        "Master instance type",
+        lambda x: x in _list_instances(),
+        default_value=_get_config_parameter(
+            config,
+            section=cluster_label,
+            parameter_name="master_instance_type",
+            default_value=DEFAULT_VALUES["master_instance_type"],
+        ),
+    )
+
+    scheduler_handler.prompt_compute_instance_type()
+
+    key_name = prompt_iterable("EC2 Key Pair Name", _get_keys(aws_region_name))
+    automate_vpc = prompt("Automate VPC creation? (y/n)", lambda x: x == "y" or x == "n", default_value="n") == "y"
+
+    vpc_parameters = _create_vpc_parameters(
+        vpc_label, aws_region_name, scheduler, scheduler_handler.max_cluster_size, automate_vpc_creation=automate_vpc
+    )
+    global_parameters = {
+        "__name__": "global",
+        "cluster_template": cluster_template,
+        "update_check": "true",
+        "sanity_check": "true",
+    }
+    aws_parameters = {"__name__": "aws", "aws_region_name": aws_region_name}
+    cluster_parameters = {
+        "__name__": cluster_label,
+        "key_name": key_name,
+        "vpc_settings": cluster_template,
+        "scheduler": scheduler,
+        "master_instance_type": master_instance_type,
+    }
+    cluster_parameters.update(scheduler_handler.get_scheduler_parameters())
+
+    aliases_parameters = {"__name__": "aliases", "ssh": "ssh {CFN_USER}@{MASTER_IP} {ARGS}"}
+    sections = [aws_parameters, cluster_parameters, vpc_parameters, global_parameters, aliases_parameters]
+
+    # We first remove unnecessary parameters from the past configurations
+    _remove_parameter_from_past_configuration(cluster_label, config, scheduler_handler.get_parameters_to_remove())
```

Comment:
Actually the parameter we remove are the one that may conflict with each other. For example, if in the previous configuration you have set scheduler on aws batch and now you wanna change, you have to remove all the parameters like min_vcpus, max_vpcus that may not be overridden in this configuration.

If we decide to flip the work and keep only the one that never conflicts, we will still need to add them somewhere whenever there is a new one. 

In this way, we have to add a parameter only if it supported only for a specific set of parameters. 

For example, if we decide to add dcv in the pcluster configure, we won't need to change anything since is does not conflict with anything